capstone/src/main/webapp/script.js

```diff
@@ -1,31 +1,153 @@
 function createMap() {
-    const map = new google.maps.Map(
-        document.getElementById('map-container'),
-        {center: {lat: 37.7749, lng: -122.4194}, zoom: 12}
-    );
-  
-    //**adding zipcode overlay */
-    map.data.loadGeoJson('zipcode-data.json');
-    //**adding precinct overlay */
-    map.data.loadGeoJson('neighborhoods.json');
-    
-    map.data.setStyle({visible: false});
-
-    const cityBorder = [
-        {lat: 37.708305, lng: -122.502691},
-        {lat: 37.708229, lng: -122.393322}
-    ]
-
-    const cityLimit = new google.maps.Polygon({
-        paths: cityBorder,
-        strokeColor: 'black',
-        strokeOpacity: 0.8,
-        strokeWeight: 2,
-        fillColor: 'black',
-        fillOpacity: 0.35
-    });
-  
+  const map = new google.maps.Map(document.getElementById('map-container'),
+      {
+        center: {lat: 37.7749, lng: -122.4194},
+        zoom: 12,
+      });
+
+  const cityBorder = [
+    {lat: 37.708305, lng: -122.502691},
+    {lat: 37.708229, lng: -122.393322},
+  ];
+
+  const cityLimit = new google.maps.Polygon({
+    paths: cityBorder,
+    strokeColor: 'black',
+    strokeOpacity: 0.8,
+    strokeWeight: 2,
+    fillColor: 'black',
+    fillOpacity: 0.35,
+  });
+
   cityLimit.setMap(map);
+
+  //* *button for centering map*/
+  const centerControlDiv = document.createElement('div');
```

Comment:
Can this div already exist in the html? I feel like this would make it clearer what the structure of the page is when you look at index.html

Reply:
I understand your concern, however I like the structure of having everything as javascript functions, so I plan to keep it as is.

Reply:
I think what Prashanti is asking for is that you make one div in the html file that is centered on the map, then in your java script functions you would just call that div by its id and then pass that to ControlPosition.LEFT_CENTER in the map.controls call

Reply:
i do not mind the style here, which ever is easier for you should be the option

Reply:
I understand, however I don't think that that will change the amount of code I have in scrips.js or make the structure of the page clearer by looking in index.html... there is a map div in html and the centerControlDiv is in the map div, so the structure of the page does not change according to line 25.

Reply:
Styling would be contained in the style sheet

Reply:
Yeah this is static styling so IMO stylewise it shouldn't be in the js script but instead in style.css

Reply:
I think it'd be clearer to have this div in the html. The reason being is that this could be creating divs but it's not immediately obvious where this div is unless you simply reference an existing div in index.html.

Reply:
Let me know if you disagree.